### DISCUSSION TITLE
Formatter should format `Float`s without digits as `x.0`

### DISCUSSION THREAD

**Discussion by inoas** [gleam-lang/gleam#1842] [Ideas & suggestions]
`1.` => formatter => `1.0` for the sake of consistency.

IMHO there is no gain in this alternative syntax versus scientific notations which would be beneficial as are underscores for grouping thouands.

**Comment by lpil** [comment]
Sounds good to me. Would you like to make an issue?

**Reply by inoas** [reply]
Can't promise but that's may be in my tiny Rust skillset ballpark :D
